CHANGELOG.md

```diff
@@ -13,8 +13,243 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 
 See [CHANGELOG - v4](https://github.com/webdriverio-boneyard/v4/blob/master/CHANGELOG.md).
 
-## v5.0.0 (2018-12-12)
+---
+
+## v5.0.0 (2018-tbd)
 
 #### :boom: Breaking Change
-* `@wdio/cli`
-    * contains now the wdio cli command
+* moved `wdio` cli command from [`webdriverio`](https://www.npmjs.com/package/webdriverio) package to [`@wdio/cli`](https://www.npmjs.com/package/@wdio/cli)
+* commands are scoped to `browser` and `element` - no selector property on commands anymore
+  ```js
+  // v4
+  browser.click('#myElem')
+  ```
+  ```js
+  // v5 and upwards
+  const elem = $('#myElem')
+  elem.click()
+  ```
+* no command chaining anymore (in standalone and wdio mode)
+  ```js
+  // v4 (standalone/async mode)
+  browser
+    .url('https://duckduckgo.com/')
+    .setValue('#search_form_input_homepage', 'WebdriverIO')
+    // ...
+  ```
+  ```js
+  // v5
+  await browser.url('https://duckduckgo.com/')
+  const elem = await browser.$('#search_form_input_homepage')
+  await elem.click()
+  ```
+* every protocol command returns `value` property instead of raw driver response
+  ```js
+  // v4
+  const result = browser.execute(() => 1 + 1)
+  console.log(result)
+  // outputs:
+  // { sessionId: '02aee149a1a421b81598ff2a3b90e33d',
+  //   value: 2,
+  //   _status: 0 }
+  ```
+  ```js
+  // v5
+  const result = browser.execute(() => 1 + 1)
+  console.log(result) // outputs: 2
+  ```
+* the `remote` and `multiremote` methods to initiate a driver instance now also start the driver session and therefor return a promise (no `init` command anymore)
+  ```js
+  // v4
+  import { remote } from 'webdriverio'
+  const driver = remote({ ... })
+  driver.init().url('https://webdriver.io').end()
+  ```
+  ```js
+  // v5
+  import { remote } from 'webdriverio'
+  const driver = await remote({ ... })
+  await driver.url('https://webdriver.io')
+  await driver.deleteSession()
+  ```
+* command changes: over the years WebdriverIO added more and more commands for different automation protocols without applying a pattern to it which resulted in having a bunch of duplication and inconsistent naming, even though the list looks exhausting, most of the commands that have changed were used internally
+    * renamed commands:
+        * `isVisible` → `isDisplayed`
+        * `isVisibleWithinViewport` → `isDisplayedInViewport`
+        * `waitForVisible` → `waitForDisplayed`
+        * `clearElement` → `clearValue`
+        * `moveToObject` → `moveTo` (element scope only)
+        * `setCookie`, `getCookie`, `deleteCookie` → `setCookies`, `getCookies`, `deleteCookies`
+        * `getElementSize` → `getSize`
+        * `source`, `getSource` → `getPageSource`
+        * `title`, `getTitle` → `getCurrentTitle`
```

Comment:
Is the 

> `title`,

 part needed here?

Reply:
yes, `title`, `getTitle` are both almost the same commands not both being replaced by `getCurrentTitle`